CHANGELOG.md

```diff
@@ -1,5 +1,132 @@
 # Release Notes
 
+## Version 1.19.0-beta.0 - 2021-04-05
+
+Features:
+
+* iso: enable Network Block Device support [#10217](https://github.com/kubernetes/minikube/pull/10217)
```

Comment:
this could be moved to the last line of features, more important features at top

Reply:
done